web/app/[workspaceSlug]/(projects)/settings/mobile-header-tabs.tsx

```
@@ -21,7 +21,7 @@ export const MobileWorkspaceSettingsTabs = observer(() => {
       {WORKSPACE_SETTINGS_LINKS.map(
         (item, index) =>
           shouldRenderSettingLink(item.key) &&
-          allowPermissions(item.access, EUserPermissionsLevel.WORKSPACE, workspaceSlug.toString()) && (
+          allowPermissions(item.access, EUserPermissionsLevel.WORKSPACE, workspaceSlug!.toString()) && (
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)